### PR TITLE
Changed force_text() to force_str()

### PR DESCRIPTION
**Pull request by smithdc1**
force_text() will be removed in Django 4.0 and is deprecated in favor of force_str()

### CODE REVIEW

File: tinymce/views.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def spell_check(request):
         if not enchant:
             raise RuntimeError("install pyenchant for spellchecker functionality")
 
-        raw = force_text(request.body)
+        raw = force_str(request.body)
```
Comment (claudep) [inline tinymce/views.py:31]: AFAIK, `json.loads` support reading bytes, so this line could be unnecessary.

File: tinymce/widgets.py [hunk 2]
```diff
@@ -75,7 +75,7 @@ def get_mce_config(self, attrs):
     def render(self, name, value, attrs=None, renderer=None):
         if value is None:
             value = ""
-        value = force_text(value)
+        value = force_str(value)
```
Comment (claudep) [inline tinymce/widgets.py:78]: This could as well be a remnant of Python 2 stuff. I would simply drop the call, as far as the test suite is happy.

**Review by JayantGoel001 — APPROVED** [review]

**Comment by Natim** [comment]
@JayantGoel001 thank you for your 2cts, I feel like @claudep comments are legit, @smithdc1 would you be willing to update your PR or shall we do it?

**Comment by smithdc1** [comment]
Sorry I've not had time this week to come back to this.

Just removing the code works, but I've not yet had chance to understand it in more detail. If you are more comfortable with the change and have time then please do carry on.

**Pull request merged**